ui.go

```diff
@@ -468,8 +468,11 @@ func (win *win) printDir(screen tcell.Screen, dir *dir, context *dirContext, dir
 
 		filename := []rune(f.Name())
 		if runeSliceWidth(filename) > maxFilenameWidth {
-			filename = runeSliceWidthRange(filename, 0, maxFilenameWidth-1)
+			var truncatePos int = (maxFilenameWidth - 1) * gOpts.fnametruncpct / 100
```

Comment:
Minor: I'm not sure if you had a reason for not using short declarations (i.e. `:=` syntax), but from what I can see most of the code uses this so it would be better to follow it:

```suggestion
			truncatePos := (maxFilenameWidth - 1) * gOpts.fnametruncpct / 100
```